docs/changes/io.ascii/16103.api.rst

```diff
@@ -0,0 +1,4 @@
+The broken parallel parser for reading ascii files is removed, it has
+been using the serial parser as a fallback since astropy v4.0.4.
+The ``parallel`` key in the ``fast_reader`` argument for reading
+ascii should not be used anymore.
```

Comment:
```suggestion
The parallel fast-reader parser for reading ASCII files has been removed.
Since astropy v4.0.4 requesting this option has issued a warning that
this option is broken and that the serial parser will be used.
The ``parallel`` key in the ``fast_reader`` argument for reading
ASCII tables is no longer available.
```